lib/test.js

```diff
@@ -12,6 +12,11 @@ const test = (suite, buildConfig = config.defaultBuildConfig, options) => {
     '--v=' + options.v,
```

Comment:
I think we need to delete the `--v` argument here, right?

Reply:
Yep sorry I had re-added it to test and make sure I wasn't crazy.  Fixed now.